qiskit/backends/ibmq/ibmqjob.py

```diff
@@ -237,38 +225,18 @@ def done(self):
         Note behavior is slightly different than Future objects which would
         also return true if successfully cancelled.
         """
-        return self.status['status'] == JobStatus.DONE
+        return self.status == JobStatus.DONE
 
     @property
     def cancelled(self):
         return self._cancelled
 
-    @property
-    def exception(self):
-        """
-        Return Exception object previously raised by job else None
-
-        Returns:
-            Exception: exception raised by job
-        """
-        if isinstance(self._exception, Exception):
-            self._status_msg = str(self._exception)
-        return self._exception
-
-    @property
-    def _is_commercial(self):
-        config = self._api.config
-        # this check may give false positives so should probably be improved
-        return config.get('hub') and config.get('group') and config.get('project')
-
     @property
     def job_id(self):
```

Comment:
should this property just be called `id`? It seems more natural to write `job.id` than `job.job_id`

Reply:
This sounds ok though I seem to remember some different meaning for `id` in the api previously but I can't find it now.